resource-watch/resource-watch
# GDPR banner added both to the app and the back office

![image](https://user-images.githubusercontent.com/545342/61132548-df144d00-a4bb-11e9-81e5-93995cbf26fb.png)

## Overview
This PR adds a GDPR banner to all pages in the platform. We use `localStorage` so that the banner is hidden once the user accepts the terms.

## Testing instructions
Check both pages from the app and the back office and see if the GDPR appears accordingly.

## [Pivotal task](https://www.pivotaltracker.com/story/show/167193001)
